src/app/page.js

```diff
@@ -15,7 +15,7 @@ const Root = async () => {
 
   const userIsAdmin = await isAdmin(user.email);
 
-  userIsAdmin ? redirect(`/admin/${user.id}/profile`) : redirect(`/user/${user.id}/profile`)
+  userIsAdmin ? redirect(`/admin/requests`) : redirect(`/user/${user.id}/profile`)
```

Comment:
PRAISE: Very nice! As soon as an admin user logs into the app, he or she goes right into the requests page! That's a great idea!

I like how you kept it so that if the user is NOT an admin, it just goes straight to the profile route.

QUESTION: Is that request route going to be called anything else? I think it's okay to just leave it as "requests". 

What do you think?